packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -15,6 +16,14 @@ const MediaIndicatorWrapper = styled.div`
   ${GEL_MINION};
   color: ${C_EBON};
   height: 2rem;
```

Comment:
This feels like im missing something, but the media indicator doesnt actually get smaller for me at <400px... The text just moved within the box

Reply:
I know.. but you can link it in the Story Promo stories and you will properly see how it changes.
Anyway, if you open the developer tools you can see that the styles change under 400px.

Reply:
![image](https://user-images.githubusercontent.com/11341355/60711342-a0572380-9f0c-11e9-9de6-c4f7c2f9550e.png)
Its still 32px tall though, which is 2rem. The height of the element needs to reduce by 12px (4 from top, 8 from bottom) so the whole outside height should be 20px not 32pz

Reply:
Fair point, if we don't change the height, doesn't matter which padding we apply that it will still fit 32px. Will double check with UX.

Reply:
I made those screenshots with Tom from UX, but np :)